javascript/reactjs-todo-oidc/.env.example

```
@@ -1,10 +1,9 @@
 API_URL=
+PORT=8443
 DEBUGGER_OFF=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DEBUGGER_OFF key should go before the PORT key

(UnorderedKey)